packages/mongodb-rag-core/src/contentStore/MongoDbSearchResultsStore.test.ts

```diff
@@ -0,0 +1,128 @@
+import { strict as assert } from "assert";
+import { assertEnvVars } from "../assertEnvVars";
+import { CORE_ENV_VARS } from "../CoreEnvVars";
+import "dotenv/config";
+import { MongoClient } from "mongodb";
+import { MONGO_MEMORY_SERVER_URI } from "../test/constants";
+import {
+  makeMongoDbSearchResultsStore,
+  MongoDbSearchResultsStore,
+  SearchResultRecord,
+} from "./MongoDbSearchResultsStore";
+
+const { MONGODB_CONNECTION_URI, MONGODB_DATABASE_NAME } =
+  assertEnvVars(CORE_ENV_VARS);
+
+describe("MongoDbSearchResultsStore", () => {
+  let store: MongoDbSearchResultsStore | undefined;
+  const searchResultRecord: SearchResultRecord = {
+    query: "What is MongoDB Atlas?",
+    results: [
+      {
+        url: "foo",
+        title: "bar",
+        text: "baz",
+        metadata: {
+          sourceName: "source",
+        },
+      },
+    ],
+    dataSources: [{ name: "source1", type: "docs" }],
+    createdAt: new Date(),
+  };
+  const uri = MONGO_MEMORY_SERVER_URI;
```

Comment:
i see you're also using memory server stuff. pls update all tests to use this like the `MongoDbPageStore` tests